src/main/java/com/google/devtools/build/lib/remote/SimpleBlobStoreActionCache.java

```diff
@@ -56,7 +66,7 @@
  * <p>Note that this class is used from src/tools/remote.
  */
 @ThreadSafe
-public final class SimpleBlobStoreActionCache extends AbstractRemoteActionCache {
+public class SimpleBlobStoreActionCache extends AbstractRemoteActionCache {
```

Comment:
Because Mockito can't mock final class in `RemoteSpawnRunnerTest.java`. `GrpcRemoteCache` was not a final class and I don't see an obvious reason to use final class here, I could be wrong.